### DISCUSSION TITLE
ComboBox with Images

### DISCUSSION THREAD

**Discussion by ponchofiesta** [emilk/egui#5279] [Q&A]
I want to draw images and text into the ComboBox options. Is this somehow possible?

**Comment by marci1175** [comment]
Hi!

I've just looked at the documentation and I came to the conclusion that there isnt a firect way as of now. You could use emotes from the font however. Im not sure if it is okay with your use case.

If I'll find a way to achieve this ill make sure to make a comment on this discussion in the future! :)
